### ISSUE TITLE
ReadableStreamBYOBReader and ReadableStreamReader disappeared with typescript 4.2.2

### ISSUE THREAD

**Issue by npenin** [microsoft/TypeScript-DOM-lib-generator#983]
I updated to typescript 4.2.2 today, but strangely, ReadableStreamBYOBReader and ReadableStreamReader disappeared from type definition

**Comment by HolgerJeromin** [comment]
ref https://github.com/microsoft/TypeScript-DOM-lib-generator/issues/886#issuecomment-664158110 and the following comment.

**Comment by npenin** [comment]
true, but now that chromium is leveraged in many browsers (vivaldi, brave, edge, chrome, ...), should we not consider to have it anyway ?

**Comment by MattiasBuelens** [comment]
FYI these types were re-added in #1362, so they will become available in a future TypeScript release. (I don't know how soon that'll happen, though.)
